etc/sample_key_invalid

```diff
@@ -0,0 +1,26 @@
+MIIEowIBAAKCAQEA4f5wg5l2hKsTeNem/V41fGnJm6gOdrj8ym3rFkEU/wT8RDtn
```

Comment:
Can we remove these key files

Reply:
These are use by token_generator tests.  These are also part of block vpc registery
Below tests will fail.

token_generator_test.go:59: 
        	Error Trace:	token_generator_test.go:59
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:60: 
        	Error Trace:	token_generator_test.go:60
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/no_sample_key", Err:0x2}
        	Test:       	TestTokenGenerator
    token_generator_test.go:63: 
        	Error Trace:	token_generator_test.go:63
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:64: 
        	Error Trace:	token_generator_test.go:64
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/no_sample_key", Err:0x2}
        	Test:       	TestTokenGenerator
    token_generator_test.go:74: 
        	Error Trace:	token_generator_test.go:74
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:75: 
        	Error Trace:	token_generator_test.go:75
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/no_sample_key", Err:0x2}
        	Test:       	TestTokenGenerator
    token_generator_test.go:85: 
        	Error Trace:	token_generator_test.go:85
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:86: 
        	Error Trace:	token_generator_test.go:86
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/sample_key_invalid", Err:0x2}
        	Test:       	TestTokenGenerator
    provider_test.go:85: {"level":"info","timestamp":"2021-04-27T15:38:39.448+0530","caller":"provider/token_generator.go:79","msg":"Entering getJWTToken","contextCredentials":{"AuthType":"IAM_ACCESS_TOKEN","DefaultAccount":false,"Region":"","IAMAccountID":"test-iks-account","ContextID":""}}